src/modules/Popup/Popup.d.ts

```diff
@@ -39,7 +39,7 @@ export interface PopupProps extends PortalProps {
   offset?: number;
 
   /** Event triggering the popup. */
-  on?: 'hover' | 'click' | 'focus';
+  on?: Array<string>;
```

Comment:
`'hover' | 'click' | 'focus' | Array<'hover' | 'click' | 'focus'>`
This should be correct as I remember